harness/determined/deploy/aws/cli.py

```diff
@@ -355,6 +378,12 @@ def handle_dump_master_config_template(args: argparse.Namespace) -> None:
                     default=None,
                     help="AWS region",
                 ),
+                Arg(
```

Comment:
rather than passing multiple comma-separated tags, can you just accept multiple `--custom-tag` or just `--tag` parameters?

Reply:
wrt:

### multiple `--custom-tag` args

I'm good with that, I'd just give priority to consistency with how things are already done in the CLI. I don't often use it myself, so if we're using flags like elsewhere, then sure. Could you link me to sample usage so I can implement that quickly?

### switching `--custom-tag` to `--tag`

We already include tags that the user has no control over. I was actually between 'custom' and 'additional' when deciding what to name this. I'd be willing to switch this to 'additional' or 'add' if that's preferable, but 'tags' by seems to imply that any defaults are overwritten imo

Reply:
multiple args will also be annoying to support in the circleci config.

Reply:
> We already include tags that the user has no control over.

This is true, but I doubt users will be offended by this fact.  If there's tags they don't control, I'd say you should have client-side checks to make sure those tags are rejected by the cli.  But regarding `--custom-tag`, the word "custom" really conveys no meaning to a user.  Same with `--add-tag`, "add" doesn't really contribute much meaning.  At the end of the day, if what gets passed to `--tag` shows up as a "tag" in AWS terminology (I'm assuming it does?) then I think `--tag` is the best option.

> Could you link me to sample usage so I can implement that quickly?

We do it for the `--include` option in cli/remote.py:
```python
            Arg(
                "-i",
                "--include",
                default=[],
                action="append",
                type=Path,
                help=command.INCLUDE_DESC
            ),
```

> multiple args will also be annoying to support in the circleci config

But `det deploy aws` is a user-facing tool, not an internal CI tool.  If it's annoying in circleci, that's not a good reason to have a worse user interface.

Reply:
Thanks for the sample. I'll see about getting it parsed as such and editing CircleCI to accommodate this.

> This is true, but I doubt users will be offended by this fact.

There's no issue with the current state of tags, imo. To be clear, I'm not suggesting getting rid of or changing those default tags whatsoever.

> have client-side checks to make sure those tags are rejected by the cli

This sounds reasonable, I'll include this.

> "custom" really conveys no meaning to a user
> "add" doesn't really contribute much meaning

I really don't agree with this. We already express that there are tags that will be set by default on the CF stack, so having _some_ indication that news tags will be _appended_ instead of _overwriting_ those default tags is meaningful and helpful.

Actually, in the context of a flag that can be set multiple times, "add-tag" or even "extra-tag" make even more sense towards the goal of making the flag's behavior self-evident.

Reply:
Ok, I'm convinced.

I like `--add-tag` then because it's sufficiently clear and shortest to type.

Reply:
Implemented and tested. ptal